API.md

```diff
@@ -110,7 +98,47 @@ If set to `True`, the cluster is destroyed on exit if there is no exception rais
 
 ### Methods
 
-#### `run_integration_tests(pytest_command, env=None)`
+#### `install_dcos_from_url(build_artifact, extra_config=None, log_output_live=False)`
+
+Installs DC/OS on the given cluster using the DC/OS advanced installation method if supported
```

Comment:
This repository follows  a practice inspired by http://rhodesmill.org/brandon/2012/one-sentence-per-line/ - one sentence per line for documentation source.